tests/Fixer/Strict/DeclareStrictTypesFixerTest.php

```diff
@@ -141,11 +155,11 @@ public function provideMessyWhitespacesCases()
     {
         return [
             [
-                "<?php declare(strict_types=1);\r\nphpinfo();",
+                "<?php\r\ndeclare(strict_types=1);\r\n\tphpinfo();",
```

Comment:
> The only thing the current fixer says is
> > Force strict types declaration in all files.

No, fixer does also provide examples (available only on CLI level). So for all the ppl who were into given style, changing it out of the box is not acceptable.
That's why we have tests. if you need to modify tests instead of only adding new tests, you change specification. Sometimes it's a bugfix, sometimes it's BC breaker.

------------

> I shall change the insertion fixer to insert on the same line

if you want to change behaviour, simply add option for that.

> the positioning fixer will stay as is.

does new fixer change sth that is also changing by current fixer? (not saying it's bad, just asking to say how to handle it if needed)

---------------

> Happy to look into it at some point, just trying to give a heads up on a possible bug here really, since this seems to be an important consideration?

if you want to, do it in separated issue, not this PR. what is needed is version before, version after, minimal code sample and rule that changed output.